src/review.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import Stars from './stars.jsx';
+import Bar from './bar.jsx';
+import CustomerReview from './customer-review.jsx';
+import Subratings from './subratings.jsx';
+
+class Review extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      'displayModal': false,
+      'itemID': 1,
+      'averageRatings': {
+        'number': 479,
+        'overall': 4.4,
+        'easeOfAssembly': 5,
+        'valueForMoney': 4,
+        'productQuality': 3,
+        'appearance': 2,
+        'worksAsExpected': 1
+      },
+      'page': {
+        'number': 1,
+        'outOf': 24
+      },
+      'customerReviews': [
+        {
+          'reviewID': 1,
+          'name': 'Carlos',
+          'date': '02/22/2021',
+          'title': 'Good set',
+          'reviewText': 'Good set',
+          'recommended': true,
+          'overall': 5,
+          'easeOfAssembly': 3,
+          'valueForMoney': 5,
+          'productQuality': 5,
+          'appearance': 5,
+          'worksAsExpected': 5
+        },
+        {
+          'reviewID': 2,
+          'name': 'JANELEN',
+          'date': '02/19/2021',
+          'title': 'Perfect for a guest bedroom',
+          'reviewText': 'Clean lines, drawers open easily and they look good.',
+          'recommended': true,
+          'overall': 5,
+          'easeOfAssembly': 4,
+          'valueForMoney': 5,
+          'productQuality': 5,
+          'appearance': 5,
+          'worksAsExpected': 5
+        }
+      ]
+    };
+  }
+
+  componentDidMount() {
+    // console.log(this.state);
```

Comment:
eventually you'll add your API calls here to render different components / update state